README.md

```diff
@@ -5,3 +5,9 @@ Mic check 1-2 ;)
 The past, the present and the future walked into a bar.
 
 ... It was tense. 
+
+***
+
+I can't believe I'm almost finished with aviation school.
+
+... This semester has really flown by.
```

Comment:
A little too *high* brow?
Maybe something a little more down to earth.

"The shovel was a ground-breaking invention."